src/plugins/telemetryTable/TelemetryTable.js

```diff
@@ -71,6 +73,21 @@ define([
             openmct.time.on('timeSystem', this.refreshData);
         }
 
+        addColunmName(telemetryObject, metadataValues) {
```

Comment:
Small grammatical suggestion - let's name this `addNameColumn`. Let's also use jsdoc to mark it private (it is only ever called internally from `initialize`.

Reply:
updated